src/samplers/multijitter.cpp

```diff
@@ -0,0 +1,159 @@
+#include <mitsuba/core/profiler.h>
+#include <mitsuba/core/properties.h>
+#include <mitsuba/core/spectrum.h>
+#include <mitsuba/render/sampler.h>
+
+NAMESPACE_BEGIN(mitsuba)
+
+/**!
+
+.. _sampler-multijitter:
+
+Correlated Multi-Jittered sampler (:monosp:`multijitter`)
+---------------------------------------------------------
+
+.. pluginparameters::
+
+ * - sample_count
+   - |int|
+   - Number of samples per pixel. This value might be slightly increased by the plugin to produce a
+     strata grid with an aspect ratio close to one. (Default: 4)
+ * - seed
+   - |int|
+   - Seed offset (Default: 0)
+ * - jitter
+   - |bool|
+   - Adds additional random jitter withing the substratum (Default: True)
+
+This plugin implements the methods introduced in Pixar's tech memo :cite:`kensler1967correlated`.
```

Comment:
IIRC this was a paper in graphics gems, would be nice to cite the peer-reviewed version. If the tech memo adds further detail, then you can add a second citation.

Reply:
Ah got it, I was just confused.